src/helper/APIHelper.ts

```diff
@@ -0,0 +1,13 @@
+const API_URL: string = "https://fakestoreapi.com/products";
+
+export const getAllItems = async () => {
```

Comment:
I normally specify the name of the function in this case because you are indeed getting products

```suggestion
export const getProducts = async () => {
```